docs/assets/matching-context.d2

```diff
@@ -0,0 +1,61 @@
+# https://d2lang.com
```

Comment:
Not important but we might decide at some point that we'd like to use separate directories for the source (d2) vs the build artifacts (SVGs).